### PR TITLE
fix: remove not used scripts in monorepo sdk

### LINKED ISSUE #998: [Bug]: SDK generated with script set:version and dependency to @o3r/workspace
### Package name

workspace

### Package version

9.5

### Reproduction steps

In an otter mono-repo run the command `ng g sdk`

### Current result

It will generate a SDK with the script `set:version` and the associated dependency to `@o3r/workspace` in the package.json

### Expected result

This script and dependency should be present only in standalone SDK

### Additional comments

_No response_

### PR DESCRIPTION
**Pull request by mrednic-1A**
## Related issues

- :bug: Fixes #998 
- :rocket: Feature #(issue)

### CODE REVIEW

File: packages/@o3r/workspace/schematics/sdk/rules/clean-standalone.rule.ts [hunk 1]
```diff
@@ -18,8 +18,11 @@ export function cleanStandaloneFiles(targetPath: string): Rule {
     tree.delete(posix.join(targetPath, '.husky', 'pre-commit'));
 
     const packageJson = tree.readJson(posix.join(targetPath, 'package.json')) as PackageJson;
-    if (packageJson.scripts?.postinstall) {
-      delete packageJson.scripts.postinstall;
+    if (packageJson.scripts) {
+      packageJson.scripts = Object.fromEntries(Object.entries(packageJson.scripts).filter(([scriptName, _]) => scriptName !== 'postinstall' && scriptName !== 'set:version'));
```
Comment (kpanot) [inline packages/@o3r/workspace/schematics/sdk/rules/clean-standalone.rule.ts:22]:
```suggestion
      const excludedScripts = ['postinstall', 'set:version'];
      packageJson.scripts = Object.fromEntries(
        Object.entries(packageJson.scripts).filter(([scriptName]) => excludedScripts.includes(scriptName))
      );
```

File: packages/@o3r/workspace/schematics/sdk/rules/clean-standalone.rule.ts [hunk 2]
```diff
@@ -18,8 +18,11 @@ export function cleanStandaloneFiles(targetPath: string): Rule {
...
+    }
+    if (packageJson.devDependencies) {
+      packageJson.devDependencies = Object.fromEntries(Object.entries(packageJson.devDependencies).filter(([depName, _]) => depName !== '@o3r/workspace'));
```
Comment (kpanot) [inline packages/@o3r/workspace/schematics/sdk/rules/clean-standalone.rule.ts:25]: The Standalone SDK should not really have dep on `@o3r/workspace`.
Why is it the case?

Comment (mrednic-1A) [reply packages/@o3r/workspace/schematics/sdk/rules/clean-standalone.rule.ts:28]: We use a bin from 'o3r/workspace' for the 'set:version'.
https://github.com/AmadeusITGroup/otter/pull/993/files#diff-7c71acfa953343012aa1685726da995972c6c12f718bb3387e60066d54228585

Comment (kpanot) [reply packages/@o3r/workspace/schematics/sdk/rules/clean-standalone.rule.ts:28]: Oooh I forgot it, never mind then

**Review by kpanot — APPROVED** [review]

**Review by matthieu-crouzet — APPROVED** [review]

**Pull request merged**